pkg/graveler/ref/merge_base_finder.go

```diff
@@ -0,0 +1,60 @@
+package ref
+
+import (
+	"container/heap"
+	"context"
+
+	"github.com/treeverse/lakefs/pkg/graveler"
+)
+
+// taken from history:
+// https://github.com/treeverse/lakeFS/blob/606bf07969c14a569a60efe9c92831f424fa7f36/index/dag/commit_iterator.go
+type CommitGetter interface {
```

Comment:
Just make it valid codedoc
```suggestion
// CommitGetter - taken from history:
// https://github.com/treeverse/lakeFS/blob/606bf07969c14a569a60efe9c92831f424fa7f36/index/dag/commit_iterator.go
type CommitGetter interface {
```